coldfront/core/project/templates/project/project_renewal/project_renewal_request_list_table.html

```diff
@@ -3,48 +3,23 @@
   <tr>
```

Comment:
I'm seeing an error trying to sort on "Project". This is the case on production too.